languages/julia/concepts/iterator-protocol/links.json

```diff
@@ -0,0 +1,2 @@
+[
+]
```

Comment:
```suggestion
[]
```

Otherwise the prettier will probably do it at some point.

Reply:
it didn't complain but good call anyway :)